packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx

```diff
@@ -297,6 +294,24 @@ export class KeytipLayerBase extends BaseComponent<IKeytipLayerProps, IKeytipLay
     }
   }
 
+  private _buildTree(): void {
+    this._keytipTree = new KeytipTree();
+    // Add regular and persisted keytips to the tree
+    for (const id in this._keytipManager.keytips) {
+      if (id) {
```

Comment:
> if (id) { [](start = 5, length = 10)

it's possible that id is undefined?

Reply:
It should not be, but the compiler enforced this =/

Reply:
would Object.keys prevent this? or what happens when you use "of"?

---
In reply to: [385317334](https://github.com/OfficeDev/office-ui-fabric-react/pull/12046#discussion_r385317334) [](ancestors = 385317334)

Reply:
of is for arrays, but I did change it to Object.keys and solve the problem :)